optaplanner-core/src/main/java/org/optaplanner/core/impl/score/ScoreUtils.java

```diff
@@ -79,6 +79,14 @@ public static double calculateTimeGradient(Number[] totalDiffNumbers, Number[] s
         return timeGradient;
     }
 
+    // TODO remove me (and all occurences) once https://issues.jboss.org/browse/PLANNER-405 is fixed
+    public static String getScoreWithUninitializedPrefix(int uninitializedVariableCount, Score score) {
```

Comment:
We 'll want the Swing UI to use this method too, as it does this code there somewhere already